tests/test_mllam_dataset.py

```diff
@@ -25,7 +27,8 @@
 )
 
 
-def test_retrieve_data_ewc():
```

Comment:
I made this into a pytest "fixture" @SimonKamuk so that I could ensure the execution order was correct. I can make a separate PR for this if you would prefer :)